### ISSUE TITLE
[Bug] stocks/fa/info - Error: Yahoo has again changed data format, yfinance now unsure which key(s) is for decryption

### ISSUE THREAD

**Issue by deeleeramone** [OpenBB-finance/OpenBBTerminal#4044]
Recreate with:

`/stocks/load aapl/fa/info`

![image](https://user-images.githubusercontent.com/85772166/214485893-698c36f2-a2b7-4422-94c9-040e91991ac2.png)

Related to this: https://github.com/ranaroussi/yfinance/issues/1329

**Comment by JerBouma** [comment]
Just fyi, this applies to anything regarding Yahoo Finance. It pretty much always includes this error message when you run something even if it does work like `/stocks/load aapl/fa/income`.

**Comment by deeleeramone** [comment]
`/Portfolio`, `Portfolio/PO`,  `/reports`, `/etf', '/crypto', basically every menu will have problems because of this.

**Comment by kajdo** [comment]
obvious - but to keep the error report complete .... affects also the sdk

**Comment by kajdo** [comment]
no guarantee that it fixes your issue - but just followed up on the issue on yfinance git and their community and devs are pretty awesome (https://github.com/ranaroussi/yfinance/issues/1329)

as said - no guarantee ... but first tests on my side were successful changing the pip package via
```
pip install git+https://github.com/ranaroussi/yfinance.git@main
```

there is an issue reported by pip because openbb has a dependency to a specific yfinance version - but it fixed my issue

**Comment by JerBouma** [comment]
Hi @kajdo,

Given the instability of `yfinance`, we're opening PRs to become less reliant on Yahoo Finance so that most functionality will continue to work despite Yahoo Finance breaking. We've noticed that market data seems to function properly 99% of the time whereas other functionality breaks so we are changing the default source. For example see #4055.

**Comment by kajdo** [comment]
@JerBouma think that's a good thing - at least for core functionalities ... that will make openbb much more stable i guess (even though the yfinance project is great, its always up to yahoo when they decide to change something)

will see if i can help out somehow

**Comment by JerBouma** [comment]
> @JerBouma think that's a good thing - at least for core functionalities ... that will make openbb much more stable i guess (even though the yfinance project is great, its always up to yahoo when they decide to change something)
> 
> will see if i can help out somehow

Yes, that's our thinking too. It is a bit unfortunate that many functionalities break as soon as `yfinance` stops working. With these changes, we are less reliant on that package making it possible to use the same version much longer. Any help is much appreciated!
